src/region/__tests__/RegionAnnotations-test.tsx

```diff
@@ -270,4 +186,50 @@ describe('RegionAnnotations', () => {
             `);
         });
     });
+
+    describe('componentDidUpdate', () => {
+        const mockRectRef = {
+            getBoundingClientRect: () => ({
+                height: 10,
+                width: 10,
+                top: 10,
+                left: 10,
+            }),
+        };
+
+        test('should call setReferenceShape current rectRef and staged exist', () => {
```

Comment:
I think there's a typo: `should call setReferenceShape **if** current rectRef and staged exist`